### PR TITLE
add Etcher to apps section

### PR DESCRIPTION
**Pull request by craig-mulligan**

### CODE REVIEW

**Comment by paulcbetts** [comment]
Great app!

**Comment by craig-mulligan** [comment]
Cheers @paulcbetts!

File: _data/apps.yml [hunk 1]
```diff
@@ -823,3 +823,14 @@
...
+  name: "Etcher"
+  description: "Burn images to SD cards & USB drives, safe & easy"
+  website: "http://www.etcher.io/"
+  repository: "https://github.com/resin-io/etcher"
+  keywords:
+    - "hardware"
+    - "developer tool"
+    - "IoT"
+  license: "Apache 2.0"
+  icon: "etcher.png"
```
Comment (kevinsawicki) [inline _data/apps.yml]: Hi, do you happen to have a 256x256 version of this icon? The new layout has them larger than before, thanks.

Comment (craig-mulligan) [inline _data/apps.yml]: Sure thing, updated @kevinsawicki.

**Pull request merged**